Sources/ZcashLightClientKit/Tor/TorClient.swift

```diff
@@ -71,6 +71,14 @@ public class TorClient {
         }
     }
 
+    public func stop() {
```

Comment:
This is more accurately a sleep function.
```suggestion
    public func sleep() {
```

Reply:
Updated